Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2015-2017 Tigera, Inc. All rights reserved.
```

Comment:
I think it should be clearer that this container is just for running tests. e.g. does it really need to be in the root directory?